decidim-proposals/spec/system/amendable/amend_proposal_spec.rb

```diff
@@ -240,10 +240,14 @@
 
           context "when the form is filled incorrectly" do
             before do
+              login_as user, scope: :user
+              visit proposal_path
+              expect(page).to have_content(proposal_title)
+              click_on "Amend"
```

Comment:
I'm not sure if this is actually doing anything. It's already defined in [line 208](https://github.com/decidim/decidim/blame/3e729da5a89f0c30b54d66daac396c6a3195fb2d/decidim-proposals/spec/system/amendable/amend_proposal_spec.rb#L208), and as far as I see locally I run the spec and not see any failure. I was checking out the history of runs to see if I can find the failing check but I can't seem to find it. Can you share it with me please? Thanks

Reply:
yeah, sure : 

There is a flacky one, https://github.com/decidim/decidim/actions/runs/7919191864/job/21619548014

The output here: 
![image](https://github.com/decidim/decidim/assets/105683/362a3dd5-6bf3-47a7-9974-54d230fc4706)

The tested interface here. Somehow, the authentication is not always happens. 

![failures_r_spec_example_groups_amend_proposal_with_amendments_enabled_when_amendment_creation_is_enabled_and_visits_an_amendable_proposal_when_the_user_is_logged_in_and_clicks_when_the_form_is_filled_incorrect_809](https://github.com/decidim/decidim/assets/105683/03963716-00fe-422d-9893-91bd3a48bd9f)

Reply:
Great, thanks for sharing it! How about if we extract this to another PR so we can backport it then?

Reply:
Already done and merged at #12473